src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.scss

```diff
@@ -27,6 +27,8 @@
     align-self: center;
     color: $body-font-color;
     margin: 0;
+    font-size: 18px;
```

Comment:
https://github.com/dotCMS/core-web/pull/525/files#diff-38afb9466705304822f75481bbc99308R30

And use rem-cal function https://github.com/dotCMS/core-web/blob/17434c3cc68d813c65e1cef20970e866bcddf013/src/styles/_mixins.scss#L3